.github/CODEOWNERS

```diff
@@ -68,10 +68,11 @@ script/check_npm_licenses.py @fmarier
 script/generate_licenses.py @fmarier
 
 # Crypto Wallets
-browser/brave_wallet/ @brave/crypto-wallets
-components/brave_wallet/ @brave/crypto-wallets
+browser/brave_wallet/ @brave/crypto-wallets-core
+components/brave_wallet/ @brave/crypto-wallets-core
 components/brave_wallet/browser/brave_wallet_provider_delegate.h @brave/crypto-wallets @brave/ios
 components/brave_wallet/resources/*.js @brave/crypto-wallets @brave/ios
+components/brave_wallet_ui/ @crypto-wallets-front-end
```

Comment:
I think this needs to be @brave/crypto-wallets-front-end

Reply:
fixed